docs/user/reference/resource-compute/list-of-commands-available-on-a-compute-resource-provisioned-by-juju/list-of-juju-introspect-macros/juju_unit_agent_name.md

```diff
@@ -0,0 +1,19 @@
+(juju-unit-agent-name)=
+# `juju_unit_agent_name`
+
+The `juju_unit_agent_name` macro returns the name of the unit 
+agent that is currently running.
+
+## Usage
+
+```python
+juju_unit_agent_name
+```
+
+Returns a string containing the agent (unit / application) name.
```

Comment:
Here and eveywhere else: This line feels like it's just duplicating the top description. I'd remove it.

Reply:
Done.